src/services.rs

```diff
@@ -275,26 +272,20 @@ impl ServiceWatcher {
     /// This intended to be called by the controller whenever it reconciles state.
     pub async fn reconcile_services(&self, apps: Vec<ModelSummary>, lattice_id: String) {
         if let Some(watcher) = self.watchers.read().await.get(lattice_id.as_str()) {
+            let wadm_client =
+                WadmClient::from_nats_client(&lattice_id, None, watcher.nats_client.clone());
```

Comment:
I don't necessarily love this, having to instantiate a new `wadm_client::Client` each time we reconcile, maybe it should be possible to do something like `.set_lattice(...)` with the `wadm_client::Client` to switch between them, since all that currently does is generate different subjects for the various endpoints

Reply:
From a code perspective, there really isn't a ton of overhead, but if there is an improvement to the client you need, let me know

Reply:
Agreed, I thinking it would be nice to store a `WadmClient` instance instead of the `async_nats::Client` in the  `Watcher` since it isn't interacting with anything other than wadm in this case, which having the ability to swap between lattices would enable.

I'm not pressed either way, it felt like not having to recreate the WadmClient every time you want to operate on a different lattice could be a nice improvement from usability perspective